addons/web_editor/tools.py

```diff
@@ -97,6 +101,13 @@ def get_video_url_data(video_url, autoplay=False, loop=False, hide_controls=Fals
             params['controls'] = 0
         if loop:
             params['loop'] = 1
+        groups = platform_match.groupdict()
+        if 'hash' in groups:
+            params['h'] = groups['hash']
+        elif 'params' in groups:
+            url_params = parse_qs(groups['params'])
```

Comment:
Hello @mart-e 

The flag here says, from what I understand, that we should be careful about the input of functions of urllib.
Could this line cause any issue if one tried to craft an URL with special params to try to access something he should not?

Basically we are trying to extract the parameters of inputs such as:
`https://player.vimeo.com/video/795669787?h=0763fdb816`

I don't really see how but I'd rather be sure.

If it's safe then a security override would be appreciated.

Thanks! 🧀